boa_engine/src/environments/compile.rs

```diff
@@ -18,15 +20,19 @@ struct CompileTimeBinding {
 /// A compile time environment maps bound identifiers to their binding positions.
 ///
 /// A compile time environment also indicates, if it is a function environment.
-#[derive(Debug, Finalize, Trace)]
+#[derive(Debug, Finalize)]
 pub(crate) struct CompileTimeEnvironment {
-    outer: Option<Gc<GcRefCell<Self>>>,
+    outer: Option<Rc<RefCell<Self>>>,
     environment_index: u32,
-    #[unsafe_ignore_trace]
```

Comment:
Was this wrong before?

Reply:
No, I just removed the `Trace` derive, since it didn't need Trace.